smoketest/Cargo.toml

```diff
@@ -12,11 +12,12 @@ codec = { package = "parity-scale-codec", version = "3.6.1", default-features =
 hex = "0.4.3"
 hex-literal = "0.4.1"
 serde = { version = "1.0.197", features = ["derive"] }
-subxt = { version = "0.37.0", features = ["substrate-compat", "unstable-light-client"] }
-subxt-macro = { version = "0.37.0" }
-subxt-metadata = { version = "0.37.0" }
-subxt-codegen = { version = "0.37.0" }
-subxt-signer = { version = "0.37.0" }
+subxt = { version = "0.38.0", features = ["substrate-compat", "unstable-light-client"] }
+subxt-macro = { version = "0.38.0" }
+subxt-metadata = { version = "0.38.0" }
+subxt-codegen = { version = "0.38.0" }
+subxt-signer = { version = "0.38.0" }
```

Comment:
Should probably update the version in `make-bindings.sh` too.

Reply:
https://github.com/Snowfork/snowbridge/pull/1327/commits/82f8baef6fc50eaaf7ae32e588e268233d9d7e1e